com.unity.render-pipelines.high-definition/Runtime/Debug/LightingDebug.cs

```diff
@@ -291,8 +309,12 @@ public bool IsDebugDisplayEnabled()
         /// <summary>Maximum number of lights against which the light overdraw gradient is displayed.</summary>
         public uint                 maxDebugLightCount = 24;
 
+        /// <summary>Exposure debug mode.</summary>
+        public ExposureDebugMode    exposureDebugMode = ExposureDebugMode.None;
         /// <summary>Exposure compensation to apply on current scene exposure.</summary>
         public float                debugExposure = 0.0f;
+        /// <summary>Whether to show tonemap curve in the histogram debug view or not.</summary>
+        public bool                 showTonemapCurve = true;
```

Comment:
This is only for exposure right? Might be nice to have a more explicit name like "showExposureTonemapCurve" otherwise it's not necessarily clear what it does from a scripting API standpoint.

Reply:
Yup good point, from UI is clear since is under the foldout, but didn't think from scripting.